resources/views/about.blade.php

```diff
@@ -73,531 +29,33 @@
-                                    <ul>
-                                        <li class="post-date"><i class="flaticon-clock m-r10"></i>7 March, 2020</li>
-                                        <li class="post-comment"><a href="javascript:void(0);"><i class="flaticon-speech-bubble"></i></a></li>
-                                        <li class="post-share"><i class="flaticon-share"></i>
-                                            <ul>
-                                                <li><a class="fa fa-facebook" href="javascript:void(0);"></a></li>
-                                                <li><a class="fa fa-twitter" href="javascript:void(0);"></a></li>
-                                                <li><a class="fa fa-linkedin" href="javascript:void(0);"></a></li>
-                                            </ul>
-                                        </li>
-                                    </ul>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    <div class="col-xl-6 wow fadeInUp" data-wow-duration="2s" data-wow-delay="0.8s">
-                        <div class="dlab-blog blog-half m-b30">
-                            <div class="dlab-media">
-                                <a href="blog-details-3.html"><img src="images/blog/default/thum4.jpg" alt=""></a>
-                            </div>
-                            <div class="dlab-info">
-                                <h5 class="dlab-title">
-                                    <a href="blog-details-3.html">Fusce sem ligula, imperdiet sed nisi sit amet, euismod posuere.</a>
-                                </h5>
-                                <p>Sed quis sagittis velit. Aliquam velit eros, bibendum ut massa et, consequat laoreet erat nam ac imperdiet.</p>
-                                <div class="dlab-meta">
-                                    <ul>
-                                        <li class="post-date"><i class="flaticon-clock m-r10"></i>7 March, 2020</li>
-                                        <li class="post-comment"><a href="javascript:void(0);"><i class="flaticon-speech-bubble"></i></a></li>
-                                        <li class="post-share"><i class="flaticon-share"></i>
-                                            <ul>
-                                                <li><a class="fa fa-facebook" href="javascript:void(0);"></a></li>
-                                                <li><a class="fa fa-twitter" href="javascript:void(0);"></a></li>
-                                                <li><a class="fa fa-linkedin" href="javascript:void(0);"></a></li>
-                                            </ul>
-                                        </li>
-                                    </ul>
-                                </div>
+                                <img src="images/video/pic2.jpg" >
```

Comment:
images always good to have alt text for seo purpose
```suggestion
                                <img src="images/video/pic2.jpg" alt="Medha's digital journey" title="Conceive, Compete, Collaborate & Contribute">
```